force-app/main/default/classes/EmailUtility.cls

```diff
@@ -0,0 +1,36 @@
+public class EmailUtility {
+    public static void toSendEmailWithTemplate(String Name, String objectType){
```

Comment:
It's better to use SObjectType standard class instead of using string. It allows to avoid going unrecognized object type names into the method. https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_class_Schema_SObjectType.htm
ex calling: toSendEmailWithTemplate('test', Contact.SObjectType);